tests/docker-compose.yml

```diff
@@ -1,85 +1,132 @@
-version: '3.8'
+version: '3.2'
```

Comment:
Is there any specific reason for degrade?

Reply:
yes, CircleCI's docker-compose version wouldn't accept higher than 3.2.